PocketKit/Sources/PocketKit/Article/Cells/ArticleComponentTextCell.swift

```diff
@@ -48,6 +57,11 @@ class ArticleComponentTextView: UITextView {
         isEditable = false
         isScrollEnabled = false
         delegate = self
+
+        interactions
+            .filter { type(of: $0) == UIContextMenuInteraction.self }
+            .forEach { removeInteraction($0) }
+        addInteraction(UIContextMenuInteraction(delegate: self))
```

Comment:
😎

Reply:
Also `$0 is UIContextMenuInteraction` could work. Not necessary by an means, but would eliminate a touch of noise in the filter condition